impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -144,21 +144,23 @@ std::shared_ptr<Texture> MakeBlurSubpass(
   return out_texture;
 }
 
-/// Calculate how much to scale down the texture depending on the blur radius.
-/// This curve was taken from |DirectionalGaussianBlurFilterContents|.
-Scalar CalculateScale(Scalar radius) {
-  constexpr Scalar decay = 4.0;   // Larger is more gradual.
-  constexpr Scalar limit = 0.95;  // The maximum percentage of the scaledown.
-  const Scalar curve =
-      std::min(1.0, decay / (std::max(1.0f, radius) + decay - 1.0));
-  return (curve - 1) * limit + 1;
-};
-
 }  // namespace
 
 GaussianBlurFilterContents::GaussianBlurFilterContents(Scalar sigma)
     : sigma_(sigma) {}
 
+/// Calculate how much to scale down the texture depending on the blur radius.
+///
+/// This value was extracted from Skia, see:
+///  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/ganesh/GrBlurUtils.cpp#L2561-L2576
+///  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/BlurUtils.h#L57
```

Comment:
No need to add another docstring.
```suggestion
// This value was extracted from Skia, see:
//  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/ganesh/GrBlurUtils.cpp#L2561-L2576
//  * https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/src/gpu/BlurUtils.h#L57
```